datasets/mx/governors/crawler.py

```diff
@@ -0,0 +1,124 @@
+import re
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+def extract_birth_place_and_date(link_governor_page, context: Context):
+    """
+    The birth place and date can be found in the page of each governor
+    in a ul tag. It is not always in the same position in the list,
+    so we will iterate over all elements and find the one that has
+    the string "Lugar y fecha de nacimiento:". If it is not found,
+    we raise a warning and return None.
+
+    :param link_governor_page: Link to a page of a specific governor
+    :param context: The context object.
+    """
+
+    response = context.fetch_html(link_governor_page)
+
+    path_to_information = '//*[@id="fContenido"]/div/div/div[2]/div/div/div/div[2]/div/div[1]/ul/li'
+
+    for element in response.xpath(path_to_information):
+        text = ' '.join(element.itertext())
+        if 'Lugar y fecha de nacimiento:' in text:
+            pattern = r'Lugar y fecha de nacimiento: ([^.]+\.) (\d{1,2}/\d{1,2}/\d{4})\.'
+            match = re.search(pattern, text)
+            if match:
+                birth_place = match.group(1)
+                birth_date = match.group(2)
+                return birth_place, birth_date
+
+    context.log.warning("Failed to identify birth place and date, link: {}".format(link_governor_page))
```

Comment:
does this currently log anything? Or is it currently always matching?

Reply:
It does log sometimes, there is some governors where this information is not present or is just the birth place or birth date. Apparently it's almost always on the older governors, the current ones have this information.

Reply:
Some examples:

- https://www.conago.org.mx/miembros/detalle/juan-jose-leon-rubio
- https://www.conago.org.mx/miembros/detalle/fernando-moreno-pena

Reply:
Ok - then I think we need to make this an info-level log so that it doesn't show up on the issues page - it's not something they're going to be fixing any time soon so it's not a transient issue.